crates/context/config/src/client/config.rs

```diff
@@ -48,125 +59,10 @@ pub struct ContextConfigClientLocalSigner {
     pub credentials: CryptoCredentials,
 }
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
-#[serde(try_from = "serde_creds::Credentials")]
-pub struct Credentials {
-    pub account_id: AccountId,
-    pub public_key: PublicKey,
-    pub secret_key: SecretKey,
-}
-
-#[derive(Clone, Debug, Deserialize, Serialize)]
-#[serde(try_from = "sn_serde_creds::Credentials")]
-pub struct SnCredentials {
-    pub account_id: String,
-    pub public_key: String,
-    pub secret_key: String,
-}
-
 #[non_exhaustive]
 #[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(untagged)]
 pub enum CryptoCredentials {
```

Comment:
do we need the `Crypto` prefix? that's implied, yeah?

Reply:
Yeah it's a leftover, it was needed before as I had Credentials and SnCredentials inside